source/docs/v3/integrations/spatie.blade.md

```diff
@@ -8,6 +8,8 @@ section: content
 
 ## **laravel-activitylog** {#laravel-activitylog}
 
+> Note: Activity's subject_id has to be an integer. E.g., to use the LogsActivity trait on a model, the model's ID has to be an integer.
```

Comment:
```suggestion
> Note: Activity's subject_id has to be an integer. For example, to use the LogsActivity trait on a model, the model's ID has to be an integer.
```

Also I think the comment should be more concrete? It just says "on a model", and "the model", but it doesn't say that this is the tenant model?

Reply:
It doesn't necessarily have to be a Tenant model, "the model" refers to the model in question (the model we want to use the trait on).

But yeah, I guess we should be concerned about the Tenant model only because that's what Tenancy provides

Reply:
The note should say that:
- *the model* needs to have an integer ID
- therefore, *the Tenant model* needs to be modified to have an integer ID in our case

And ideally, mention that extra security measures are recommended with integer IDs for tenants, because the IDs become enumerable. uuids are safe against enumeration attacks.

Reply:
Updated the note
https://github.com/stancl/tenancy-docs/compare/526f4eb1b8b42d177402e3c4058a45eb8692dd4e..Add-note-to-laravel-activitylog-integration

Reply:
If this is resolved, you can close the review.